terragrunt/env/staging/scan_queue/state-machines/scan-queue-lock-cleanup.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
Are the state machine configs environment specific or are they the same for both envs?  Reason I ask is because if they're the same (or mostly the same) we could move them into the `terragrunt/aws/scan_queue` folder so we didn't have to keep two copies.

Also, clearly I defer to the Head of State Machines as to what's best 👑

Reply:
Good point! i'll move them to the aws folder when i do the `scan-queue` PR since it has a dep on API outputs.